cmd/ingester/main.go

```diff
@@ -36,6 +37,12 @@ func main() {
 		&schemaConfig, &ingesterConfig)
 	flag.Parse()
 
+	hook, err := promrus.NewPrometheusHook()
+	if err != nil {
+		log.Fatalf("Error initializing promrus: %v", err)
+	}
```

Comment:
Maybe add a `MustNewPrometheusHook` variant later?

Reply:
Good idea. Let's keep it DRY. I didn't know that `MustNew*` was actually so idiomatic, thanks for pointing it out.

Reply:
Done in #573.